pmoves/tools/tz-cli/tz_cli/cli.py

```
@@ -0,0 +1,741 @@
+            sys.exit(1)
+
+        # Remove variant
+        del config["functions"][function_name][variant_key]
+
+        # Write back to file
+        with open(DEFAULT_CONFIG_PATH, "w") as f:
+            toml.dump(config, f)
+
+        success_msg(f"Removed variant '{variant_name}' from function '{function_name}'")
+
+    except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)